src/screens/ContactScreen/index.js

```diff
@@ -129,14 +139,15 @@ const ContactScreen = ({navigation}) => {
       username={item.username}
       followed={extendedState.followed}
       userid={item.user_id}
+      isAnon={sourceScreen === ANONYMOUS}
       onPress={() => handleSelected(item)}
     />
   );
   // }
   const handleSelected = (value) => {
-    const copyFollowed = [...followed];
-    const copyUsername = [...usernames];
-    const copyUsers = [...selectedUsers];
+    const copyFollowed = sourceScreen === ANONYMOUS ? [] : [...followed];
+    const copyUsername = sourceScreen === ANONYMOUS ? [] : [...usernames];
+    const copyUsers = sourceScreen === ANONYMOUS ? [] : [...selectedUsers];
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This block of code has been modified to handle different selection scenarios based on the source screen. If the source screen is `ANONYMOUS`, it creates new empty arrays; otherwise, it copies the existing arrays. This logic seems fine, but make sure these new additions are used properly throughout the code.
```diff
+ isAnon={sourceScreen === ANONYMOUS}
+ onPress={() => handleSelected(item)}
+ const copyFollowed = sourceScreen === ANONYMOUS ? [] : [...followed];
+ const copyUsername = sourceScreen === ANONYMOUS ? [] : [...usernames];
+ const copyUsers = sourceScreen === ANONYMOUS ? [] : [...selectedUsers];
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->